ztp/Makefile

```diff
@@ -1,17 +1,13 @@
-POLICYGEN_DIR := ./ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator
+.PHONY: ci-job build-policygen checkSourceCrs build-policygen-kustomize
 
-export GOFLAGS := $(if $(GOFLAGS),$(GOFLAGS),-mod=vendor)
+ci-job: build-policygen checkSourceCrs build-policygen-kustomize
 
-.PHONY: ci-job govet gotest checkSourceCrs
-ci-job: govet gotest checkSourceCrs
-
-govet:
-	@echo "ZTP: Running 'go vet'"
-	go vet $(POLICYGEN_DIR)/...
-
-gotest:
-	@echo "ZTP: Running 'go test'"
-	go test $(POLICYGEN_DIR)/...
+build-policygen:
+	@echo "ZTP: Build policy generator and run test"
+	$(MAKE) -C ./policygenerator all
 
 checkSourceCrs:
 	$(MAKE) -C ./source-crs-builder check
+
+build-policygen-kustomize:
+	$(MAKE) -C ./policygenerator-kustomize-plugin build
```

Comment:
File should ideally have a new-line at the end

Reply:
okay